Source/Scene/PointCloud3DTileContent.js

```diff
@@ -392,37 +451,32 @@ define([
 
         // If points are not batched and there are per-point properties, use these properties for styling purposes
         var styleableProperties;
-        if (!defined(batchIds) && defined(batchTableBinary)) {
+        if (!hasBatchIds && defined(batchTableBinary)) {
             styleableProperties = Cesium3DTileBatchTable.getBinaryProperties(pointsLength, batchTableJson, batchTableBinary);
-
-            // WebGL does not support UNSIGNED_INT, INT, or DOUBLE vertex attributes. Convert these to FLOAT.
-            for (var name in styleableProperties) {
-                if (styleableProperties.hasOwnProperty(name)) {
-                    var property = styleableProperties[name];
-                    var typedArray = property.typedArray;
-                    var componentDatatype = ComponentDatatype.fromTypedArray(typedArray);
-                    if (componentDatatype === ComponentDatatype.INT || componentDatatype === ComponentDatatype.UNSIGNED_INT || componentDatatype === ComponentDatatype.DOUBLE) {
-                        oneTimeWarning('Cast pnts property to floats', 'Point cloud property "' + name + '" will be casted to a float array because INT, UNSIGNED_INT, and DOUBLE are not valid WebGL vertex attribute types. Some precision may be lost.');
-                        property.typedArray = new Float32Array(typedArray);
-                    }
-                }
-            }
         }
 
         content._parsedContent = {
             positions : positions,
             colors : colors,
             normals : normals,
             batchIds : batchIds,
-            styleableProperties : styleableProperties
+            styleableProperties : styleableProperties,
+            draco : {
```

Comment:
This `draco` property should probably remain undefined unless the extension is used.